README.md

```diff
@@ -1,6 +1,6 @@
-# gatsby-contentful-starter
+# Contentful Gatsby Starter Blog
 
-Gatsby [Contentful](https://www.contentful.com) starter for creating a blog
+Create a [Gatsby](http://gatsbyjs.com/) blog powered by [Contentful](https://www.contentful.com). This is a simplified version of the [Gatsby Contentful Starter](https://github.com/contentful-userland/gatsby-contentful-starter) maintained by our Community.
```

Comment:
the wording _could_ possibly cause mild confusion here. Please rewrite to "which is maintained by our Community." at the end.